tests/runRule.ts

```diff
@@ -66,7 +66,9 @@ function runRuleOnFile(Rule: any, file: string): LintError[] {
   let failures: tslint.RuleFailure[];
 
   if ((rule as tslint.Rules.TypedRule).applyWithProgram) {
-    const program = ts.createProgram([file], ts.getDefaultCompilerOptions());
+    const compilerOptions = ts.getDefaultCompilerOptions();
+    compilerOptions.target = ts.ScriptTarget.ES2017;
```

Comment:
@vilchik-elena I think we should read this from our project's properties, it's not specific to the tests. Especially if we extract and centralize this